src/App.tsx

```diff
@@ -1,61 +1,137 @@
 import './App.scss';
+import { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { TodoList } from './components/TodoList';
+import { User } from './type/User';
+import { Todo } from './type/Todo';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
 
 export const App = () => {
+  const [userEntered, setUser] = useState('');
+  const [title, setTitle] = useState('');
+  const [userError, setUserError] = useState(false);
+  const [titleError, setTitleError] = useState(false);
+  const [addTodo, setAddTodo] = useState(todos);
```

Comment:
naming is confusing
`addTodo` is more suitable for a function that adds a todo to the list
```suggestion
  const [todoList, setTodoList] = useState(todos);
```